app/models/order_detail.rb

```diff
@@ -926,4 +925,8 @@ def set_reconciled_at
     self.reconciled_at ||= Time.current
   end
 
+  def problem_description_key
+    time_data.problem_description_key || (:price_policy_missing if price_policy.blank?)
```

Comment:
Something kind of bothers me about how this reads, but I'm not quite sure how to improve it.

Maybe something like this (unsure if this syntax actually works):
```
[
  time_data.problem_description_key,
  :price_policy_missing if price_policy.blank?,
].first
```